include/boost/regex/concepts.hpp

```diff
@@ -438,16 +437,11 @@ struct BaseRegexConcept
       sub_diff_type diff = m_sub.length();
       ignore_unused_variable_warning(diff);
       // match_results tests:
-      typedef typename match_results_type::value_type mr_value_type;
```

Comment:
I appreciate that these are unused, but the whole point of these is to verify that the match_result nested typedefs exist.  Removing them simply removes the test.

Reply:
Makes sense.  I'll undo those changes.  Does this comment apply to line 367 as well?

Reply:
Yes, everything in that file.